### PR TITLE
Feat 24 incorporate uswds

### PR DESCRIPTION
**Pull request by jhackshaw-dds**
Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist
I have…
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.
~~review and update SSAG documentation if needed.~~

## Summary of Changes
This pull request…
- #24 
- Bring in USWDS npm dependency
    - Configure react to compile SASS as part of build process
- Add a header using the uswds
- Setup reusable unit testing utilities with RTL
- Setup absolute imports from src/
- Add and POC react-router

## Testing
To verify the changes proposed in this pull request…
1. Run unit tests
2. Start development server and view changes in browser

## Screenshots
Screen Shot 2020-01-29 at 12 03 27 PM

---

Screen Shot 2020-01-29 at 12 04 10 PM

### CODE REVIEW

File: frontend/src/components/Header/__tests__/header.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    props.onClose.mockReset();
+  });
+
+  it("matches snapshot @snapshot", () => {
+    const { asFragment } = render(<HeaderNav {...props} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("adds visible class when open", () => {
+    const { container } = render(<HeaderNav {...props} />);
```
Comment (SCasarottoDDS) [inline frontend/src/components/Header/__tests__/header.spec.tsx]: Should this have `isOpen` or `isOpen={true}` and not reply on the default?

**Review by SCasarottoDDS — COMMENTED** [review]
Overall this is looking good so far.

File: frontend/src/components/Header/__tests__/header.spec.tsx (hunk 1 above)
Comment (jhackshaw-dds) [reply frontend/src/components/Header/__tests__/header.spec.tsx]: Agreed, makes it more clear what the test is doing. Fixed.

**Review by vcastro-DDS — APPROVED** [review]
Reviewed the code and visited stage.solo.code.mil

The page reflects the functionality of the code.

**Pull request merged**